README.md

```diff
@@ -149,6 +155,32 @@ subtract_omit = */.tox/*
 this will result in `*/.tox/*` not being `omit`ted (`*/.tox/*` is among the
 defaults provided by `covdefaults`).
 
+#### coverage for installed libraries
+
+By default ``covdefaults`` assumes that you want to track coverage
+in your local source tree, code that is version controlled. For libraries
+it's better idea to detect coverage on your installed version, and map that
+back to the source files. You can enable tracking installed libraries via:
```

Comment:
I disagree with this opinion (and I know that I'm wrong and I don't care) so I'd rather not put this as a _fact_ here